### PR TITLE
Reduce gif size

### LINKED ISSUE #188: remove gif in readme?
Personally I find it distracting (especially since it's the first thing we see on datawizard homepage)

### PR DESCRIPTION
**Pull request by etiennebacher**
Close #188. @DominiqueMakowski feel free to modify as you want, for now it looks like this:

image

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -22,11 +22,18 @@ library(datawizard)
 [![total](https://cranlogs.r-pkg.org/badges/grand-total/datawizard)](https://cranlogs.r-pkg.org/) [![status](https://tinyverse.netlify.com/badge/datawizard)](https://CRAN.R-project.org/package=datawizard)
 
 
-***Hockety pockety wockety wack, prepare this data forth and back*** :sparkles:
+<img src='https://media.giphy.com/media/VcizxCUIgaKpa/giphy.gif' width="200" align = "right"/>
 
-`datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis.
+***:sparkles: Hockety pockety wockety wack, prepare this data forth and back***
```
[the remark this answers is not in the document]
Comment (etiennebacher) [reply README.Rmd:27]: ah ok I wasn't sure if you wanted to include it :sweat_smile:

[the remark this answers is not in the document]
Comment (etiennebacher) [reply README.Rmd:27]: but you want to keep the gif, right?

**Comment by IndrajeetPatil** [comment]
I am really not sure about having that poem upfront, or even having it in the README at all.

I'd also like for other weight in on this.

**Comment by strengejacke** [comment]
> I think the poem is a little confusing...

It's a "rime richissime".

**Comment by etiennebacher** [comment]
what about the gif?

File: README.Rmd [hunk 2]
```diff
@@ -22,11 +22,19 @@ library(datawizard)
...
-<img src='https://media.giphy.com/media/VcizxCUIgaKpa/giphy.gif' height="150" />
+<!-- ***Hockety pockety wockety wock, messy data is in shock*** -->
+
+<!-- ***Hockety pockety wockety woss, you can cite i-it from JOSS*** <sup>(soon)</sup> -->
+
+<!-- ***Hockety pockety wockety wass, datawizard saves your ass! :sparkles:*** -->
+
+Most courses and tutorials about statistical analysis and modelling focus on the different methods and assume that you have a cleaned and tidy dataset at your disposal. In practice however, it is very rare to obtain the dataset you're going to analyze without cleaning values, creating new columns, reshaping the dataset, or transforming some variables. It is therefore essential to have some tools to perform these tedious tasks. 
+
+Here comes `datawizard`, a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.
```
Comment (bwiernik) [inline README.Rmd:37]:
```suggestion
`datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.

Most courses and tutorials about statistical modeling focus assume that you are working with a clean and tidy dataset. In practice, however, a major part of doing statistical modeling is preparing your data--cleaning up values, creating new columns, reshaping the dataset, or transforming some variables. `datawizard` provides easy to use tools to perform these common, critical, and sometimes tedious data preparation tasks. 
```

**Review by bwiernik — CHANGES_REQUESTED** [review]

File: README.Rmd (hunk 2 above)
Comment (etiennebacher) [reply README.Rmd]: done

File: README.Rmd [hunk 3]
```diff
@@ -22,11 +22,19 @@ library(datawizard)
...
+<!-- ***:sparkles: Hockety pockety wockety wack, prepare this data forth and back*** -->
 
-<img src='https://media.giphy.com/media/VcizxCUIgaKpa/giphy.gif' height="150" />
+<!-- ***Hockety pockety wockety wock, messy data is in shock*** -->
+
+<!-- ***Hockety pockety wockety woss, you can cite i-it from JOSS*** <sup>(soon)</sup> -->
+
+<!-- ***Hockety pockety wockety wass, datawizard saves your ass! :sparkles:*** -->
+
+`datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.
```
Comment (IndrajeetPatil) [inline README.Rmd:35]:
```suggestion
`{datawizard}` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.
```

Comment (IndrajeetPatil) [inline README.Rmd:35]:
```suggestion
`datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/easystats/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.
```

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]

File: README.Rmd (hunk 3 above)
Comment (etiennebacher) [reply README.Rmd]: done

**Review by IndrajeetPatil — APPROVED** [review]
Thanks!

**Pull request merged**